dentapp/src/components/loginButton.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Styled from "styled-components";
+
+function LoginButton() {
+  const handleClick = () => {
+    console.log("The button was clicked");
+  };
+
+  const Loginbutton = Styled.button`
+  background-color: #DBF8FA;
+  color:  #00ADC1;
+  border: none;
+  border-radius: 60px;
+  font-family: Roboto;
+  letter-spacing: 0.0357143em;
+  text-transform: uppercase;
+  width: 311px;
+  height: 53px;
+  `;
```

Comment:
Let's call it 'Button', if you see, the function component and the styled component have the same name, just the uppercase difference